tests/acceptance/test_nominal.py

```diff
@@ -82,6 +82,17 @@ def test_nominal(defer, worker):
     assert "Print something to stdout" in stdout
     assert stderr.count("Exception: This should fail") == 2
 
+    defer("multiple_exception_failures")
+    stdout, stderr = worker()
+    assert (
+        stdout
+        == """Launching a worker on all queues
+Try 0
+Try 1
+Try 2
+"""
+    )
```

Comment:
Shouldn't we also test stderr, to make sure it failed as expected?

Reply:
Indeed, costs nothing, and it's highly relevant.

Reply:
@emmanuelleq How about you take care of this fix ? I'll be available to help, if needed!